ansible/roles/aws_ebs_volumes/tasks/main.yml

```diff
@@ -1,54 +1,57 @@
 ---
+  - name: Create and mount filesystems
+    become: true
+    block:
 
-  - name: Create ext4 filesystems for /var/lib/docker, /tmp, persistent volumes
-    async: 200
-    register: filesystem_creations
-    poll: 0
-    filesystem:
-      fstype: ext4
-      dev: '{{ item }}'
-      opts: -E lazy_itable_init
-    with_items:
-      - "{{ var_lib_docker_block_device }}"
-      - "{{ tmp_block_device }}"
-      - "{{ persistent_vol_block_device }}"
+    - name: Create ext4 filesystems for /var/lib/docker, /tmp, persistent volumes
```

Comment:
Same here, is the formatting change intentional?

Reply:
If referring to the tabbing out, see comment above.